### PR TITLE
racelab: Add version 3.5.0

### PR DESCRIPTION
**Pull request by scowalt**
- [x] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

Racelab provides overlays for racing games. The package `actools` is also in this bucket, which is another application meant to compliment racing games.

I've been running this manifest in my scoop bucket for a bit: https://github.com/scowalt/scoop-apps

### CODE REVIEW

File: bucket/racelab.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "version": "3.5.0",
+    "license": "Proprietary",
+    "description": "Racelab is a third party overlay for iracing, Assetto Corsa Competizione, and Formula 1, that improves your race experience.",
+    "homepage": "https://racelab.app",
+    "url": "https://update.racelab.app/RacelabApps-3.5.0%20Setup.exe",
+    "hash": "421380d63c9de498818bfbe0b1077987e9d756d426a12808b1aa9ed4f1148373",
+    "installer": {},
+    "uninstaller": {
+        "script": "Invoke-Expression -Command \"$env:LOCALAPPDATA\\RacelabApps\\Update.exe --uninstall\""
+    },
```
Comment (scowalt) [inline bucket/racelab.json:11]: I want to double-check that these scripts don't need admin mode

File: bucket/racelab.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "3.5.0",
+    "license": "Proprietary",
+    "description": "Racelab is a third party overlay for iracing, Assetto Corsa Competizione, and Formula 1, that improves your race experience.",
```
Comment (Calinou) [inline bucket/racelab.json:4]:
```suggestion
    "description": "Third-party overlay for iRacing, Assetto Corsa Competizione, and Formula 1 that improves your race experience",
```

File: bucket/racelab.json (hunk 1 above)
Comment (scowalt) [reply bucket/racelab.json:11]: I just validated that the installer and uninstaller run without admin.

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

**Comment by scowalt** [comment]
Thanks for the quick review + merge!

I have a few more game-related packages from my personal bucket that I'll migrate over too

**Pull request merged**